react/src/components/Form/SelectField.tsx

```diff
@@ -67,8 +67,10 @@ function SelectField({
               multiValue: (provided, state) => ({
                 ...provided,
                 border: "1px",
-                borderColor: colorIsBright(state.data?.color) ? "gray.300" : state.data?.color,
-                color: colorIsBright(state.data?.color) ? "black" : "white",
+                borderColor: colorIsBright(state.data?.color ?? "#fff")
```

Comment:
I'm don't fully understand what this does. If `state.data.color` is undefined, you pass `#fff` to `colorIsBright`? How is `#fff` a valid color code? Also if `#fff` stands for white color, it will be assessed as bright color anyways.

Reply:
@pylipp this is actually needed as when we have multiple choice selects it assumes you set the default background color, causing issues, so as a quick fix I added the default colors (background color to grey and font color to white)